src/index.tsx

```diff
@@ -26,131 +27,17 @@ import registerExtensions from './registerExtensions';
 import * as serviceWorker from './serviceWorker';
 import './styles.css';
 
-/*
-What does this middleware do?
-
-It allows us to dispatch actions to manipulate the redux state
-in the usual way, but if the action has a persistKey field,
-then this middleware redirects the action to the server for
-persistence. Once the server has applied the action to the live
-feed, the new messages on the feed will propagate back to the
-client side (there is a feed listener) and at that time the action
-will actually be played out on the redux state (because the
-persistKey will have been stripped).
-*/
-const persistStateMiddleware: Middleware = (api: MiddlewareAPI) => (next: Dispatch<AnyAction>) => (action: any) => {
```

Comment:
We know longer need the persistStateMiddleware! It has been replaced by the `useFeedReducer`, which accomplishes the same goal - but without redux.